spec/requests/organisation_relationships_controller_spec.rb

```diff
@@ -40,7 +40,7 @@
           it "shows a table of stock owners" do
             expected_html = "<table class=\"govuk-table\""
             expect(response.body).to include(expected_html)
-            expect(response.body).to include(stock_owner.name)
+            expect(CGI.unescapeHTML(response.body)).to include(stock_owner.name)
```

Comment:
This was causing flaky tests cause out organisation names can have special characters in them now